src/Controls/src/Xaml/Hosting/AppHostBuilderExtensions.cs

```diff
@@ -109,6 +109,10 @@ public static IMauiHandlersCollection AddMauiControlsHandlers(this IMauiHandlers
 			handlersCollection.AddHandler(typeof(Frame), typeof(Handlers.Compatibility.FrameRenderer));
 #endif
 
+#if WINDOWS || MACCATALYST
+			handlersCollection.AddHandler(typeof(ContextFlyout), typeof(ContextFlyoutHandler));
```

Comment:
This code appears to work fine on `iOS13+`. Not sure if there was a specific reason why we aren't letting this code just run it's course on iOS13. I created a tracking issue [here](https://github.com/dotnet/maui/issues/9445) that we can address after we're API completely for NET7 since it shouldn't take any API changes to enable the handlers for iOS